cross/sickchill/Makefile

```diff
@@ -1,14 +1,10 @@
 PKG_NAME = SickChill
-PKG_VERS = 2021.03.10-1
+PKG_VERS = 2021.03.28-3
```

Comment:
Is this supposed to match the PKG version?

Reply:
Ideally yes but the other way around? https://github.com/SynoCommunity/spksrc/pull/4161#discussion_r593827730

Reply:
This is also used to download the tarball

Reply:
The shell date that was there had no dots, so when I tried changing version to be set with the dots properly I could not update in syno gui because it thought I was downgrading. I would rather have the dots.

Reply:
Sorry it's too late. 2021 is simply smaller than 20210329 there is nothing we can do now. My hint/warning could have been clearer. You probably wanted `date +%Y.%m.%d`